nerves-hub/setup/firmware.md

```diff
@@ -20,3 +20,25 @@ Firmware can also be signed while publishing:
 mix nerves_hub.firmware publish --key devkey
 ```
 
+## Delta Updates
+
+With Delta updates, NervesHub can compare two binary files and only update the pieces of the firmware that have changed. This can reduce the size and cost of OTA updates by up to 95%. In order use Delta Updates, you need to do the following:
+
+Be using a fwup >= 1.6.0. If your nerves system is based on nerves_system_br >= 1.11.2 then this will be fine. If not, you may need to upgrade your nerves system.
+
+Be using 	nerves_hub_link >= 0.10.0 or nerves_hub_link_http >= 0.9.0. These versions have not been released yet so the best thing to do for now is to install from the main branch. In your mix.exs file:
+
+```bash
+{:nerves_hub_link_http, github: "nerves-hub/nerves_hub_link_http", branch: "main"}
+{:nerves_hub_link, github: "nerves-hub/nerves_hub_link", branch: "main"}
+```
```

Comment:
```suggestion
* `nerves_hub_link >= 0.10.0` or `nerves_hub_link_http >= 0.9.0`
```

We can release the versions to match the requirement. Looks like the work is already in place?

Reply:
Yeah, if you can release them we should be good!